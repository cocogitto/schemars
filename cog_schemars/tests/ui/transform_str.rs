use cog_schemars::JsonSchema;

#[derive(JsonSchema)]
#[cog_schemars(transform = "x")]
pub struct Struct;

fn main() {}
