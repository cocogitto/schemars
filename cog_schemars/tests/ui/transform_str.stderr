error: Expected a `fn(&mut Schema)` or other value implementing `cog_schemars::transform::Transform`, found `&str`.
       Did you mean `#[cog_schemars(transform = x)]`?
 --> tests/ui/transform_str.rs:4:28
  |
4 | #[cog_schemars(transform = "x")]
  |                            ^^^
